CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 ## Changelog
 
 ### 2.0.3 (Next)
-
+* [#101](https://github.com/mongoid/mongoid-locker/pull/101): Replace the $where operator with the $expr operator - [@amanfrinati](https://github.com/amanfrinati).
 * [#102](https://github.com/mongoid/mongoid-locker/pull/102): Re-added danger, removed Travis-CI config, extracted RuboCop - [@dblock](https://github.com/dblock).
 * [#104](https://github.com/mongoid/mongoid-locker/pull/104): Fixed code coverage - [@dblock](https://github.com/dblock).
-* [#101](https://github.com/mongoid/mongoid-locker/pull/101): Replace the $where operator with the $expr operator - [@amanfrinati](https://github.com/amanfrinati).
-* Your contribution here.
+* [#105](https://github.com/mongoid/mongoid-locker/pull/105): Upgraded supported mongoid version to version 9.0 - [@saisrinivasan](https://github.com/SairamSrinivasan)
```

Comment:
Change to "Added support for Mongoid 9.0". Don't forget the period at the end or danger will complain.